projects/cashmere/src/lib/app-switcher/app-switcher.component.html

```diff
@@ -12,9 +12,9 @@ <h4>Applications
     <ng-container *ngIf="applications?.length > 0">
         <div class="apps">
             <div class="app" *ngFor="let app of applications">
-                <a class="thumbnail" [title]="app.Description" [href]="app.ServiceUrl" target="_blank">
-                    <img class="thumbnail-img" [height]="iconHeight" [src]="app.Icon" [ngStyle]="{'max-width': iconHeight + 'px'}">
-                    <div class="title">{{app?.FriendlyName | ellipsis:18}}</div>
+                <a class="thumbnail"  [title]="app.Description" [attr.href]="linkIfNotMe(app)" target="_blank">
```

Comment:
`linkIfNotMe` is nice!  I was just going to ask if we should disable the link for the current app, but I see you're way ahead of me 👍